service/login_record_service.py

```diff
@@ -28,3 +28,8 @@ def recieved(self, ip: str) -> dict:
             return received_homes[0]
         else:
             return {}
+
+    def is_exist(self, ip: str) -> dict:
```

Comment:
アノテーションが違うからこれだけ直して欲しい！
```suggestion
    def is_exist(self, ip: str) -> bool:
```